<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntradaSalida</class>
 <widget class="QDialog" name="EntradaSalida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cantidad</string>
     </property>
     <property name="buddy">
      <cstring>lineEditCantidad</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Motivo</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxMotivo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBoxMotivo">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditCantidad">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCAncelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLineEdit" name="lineEditDescripcion"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditCantidad</tabstop>
  <tabstop>comboBoxMotivo</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>pushButtonAceptar</tabstop>
  <tabstop>pushButtonCAncelar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCAncelar</sender>
   <signal>clicked()</signal>
   <receiver>EntradaSalida</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
